input/ez.raml

```diff
@@ -2817,110 +2817,110 @@ version: 2.5
                 get:
                     displayName: Load Role draft
-                    description: Loads a Role draft by original Role ID
+                    description: Loads a Role draft by original Role ID.
                     headers:
                         Accept:
-                            description: If set the user list returned in XML or JSON format
+                            description: If set, the User list returned in XML or JSON format.
                             example:  |
                                 application/vnd.ez.api.Role+xml
                                 application/vnd.ez.api.Role+json
                         If-None-Match:
                             description: ETag
                     responses:
                         200:
-                            description: OK - Role draft by original Role ID
+                            description: OK - Role draft by original Role ID.
                             body:
                                 application/vnd.ez.api.Role+xml:
                                     type: Role
                                     example: !include examples/user/roles/id/draft/GET/Role.xml.example
                         401:
-                            description: Error - the user has no permission to read roles
+                            description: Error - the user has no permission to read Roles
                         404:
                             description: Error - there is no draft or Role with the given ID
                 patch:
                     displayName: Update Role draft
-                    description: Updates a Role draft. PATCH or POST with header X-HTTP-Method-Override PATCH
+                    description: Updates a Role draft. PATCH or POST with header X-HTTP-Method-Override PATCH.
                     headers:
                         Accept:
-                            description: If set the updated Role is returned in XML or JSON format
+                            description: If set, the updated Role is returned in XML or JSON format.
                             example:  |
                                 application/vnd.ez.api.Role+xml
                                 application/vnd.ez.api.Role+json
                         Content-Type:
-                            description: The RoleInput schema encoded in XML or JSON
+                            description: The RoleInput schema encoded in XML or JSON format.
                             example:  |
                                 application/vnd.ez.api.RoleInput+json
                                 application/vnd.ez.api.RoleInput+xml
                         If-Match:
-                            description: ETag Causes to patch only if the specified ETag is the current one. Otherwise a 412 is returned.
+                            description: Causes to PATCH only if the specified ETag is the current one. Otherwise a 412 is returned.
```

Comment:
```suggestion
                            description: Performs a PATCH only if the specified ETag is the current one. Otherwise a 412 is returned.
```